src/server/registry/referrers.go

```diff
@@ -75,29 +84,69 @@ func (r *referrersHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		lib_http.SendError(w, err)
 		return
 	}
-
 	// Build index manifest from accessories
 	mfs := make([]ocispec.Descriptor, 0)
 	for _, acc := range accs {
-		accArt, err := r.artifactManager.GetByDigest(ctx, repository, acc.GetData().Digest)
+		accArtDigest := acc.GetData().Digest
+		accArt, err := r.artifactManager.GetByDigest(ctx, repository, accArtDigest)
 		if err != nil {
 			lib_http.SendError(w, err)
 			return
 		}
-		mf := ocispec.Descriptor{
+		// whether get manifest from cache
+		fromCache := false
```

Comment:
the fromCache confuses me, the exactly meaning of this variable here is to success read manifest from cache.  It is not pointing whether to get manifest from cache

Reply:
When user not enable cache layer config  or read manifest from cache failed, it need pull manifest which would be `not fromCache`.
Only when Cache enabled and successfully read manifest from redis, fromCache would be true.